src/main/java/io/github/aquerr/steamwebapiclient/response/WorkShopQueryResponse.java

```diff
@@ -11,117 +15,342 @@
 @Data
 public class WorkShopQueryResponse implements SteamWebApiResponse {
 
+    /**
+     * The response.
+     */
     @JsonProperty("response")
     private QueryFilesResponse response;
 
     @Data
     public static class QueryFilesResponse {
+
+        /**
+         * The pointer to next page.
+         */
         @JsonProperty("next_cursor")
         private String nextCursor;
+
+        /**
+         * The list of found workshop items.
+         */
         @JsonProperty("publishedfiledetails")
-        private List<PublishedFileDetails> publishedFileDetails;
+        private List<PublishedFileDetails> publishedFileDetails = new ArrayList<>();
```

Comment:
It should be like that. We should always return an empty list if we do not find any results.